### DISCUSSION TITLE
Alist 3.11迅雷云盘（非专家版）经常出错

### DISCUSSION THREAD

**Discussion by KennyZKY** [AlistGo/alist#3560] [Q&A]
Alist版本3.11，迅雷已挂载成功，驱动为迅雷（非专家版），infuse连接alist也能看，但alist迅雷云盘经常性挂载失灵。状态显示为：Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": dial tcp: lookup xluser-ssl.xunlei.com on 127.0.0.1:53: no such host
有时迅雷云盘会自己恢复正常，周而复始。阿里云盘挂载正常。

**Comment by youtehub** [comment]
挂专家版不就行了，我一直在用 专家版很稳定, 只需要输入，用户名，密码，还有验证码签名 就行了，用下载软件下载东西最高可以达到 45M/S，千兆宽带下哈，用的  BitComet (比特彗星) (64-bit) 1.98
阿里网盘最近倒是弃用了，主要是问题多不稳定，TG 群里封的阿里云盘被封的人还不少
![image](https://user-images.githubusercontent.com/18629921/220707847-011a36b4-d6e9-4471-930e-496110c96f88.png)

**Reply by KennyZKY** [reply]
请问验证码签名怎么弄的？

**Reply by anwen-anyi** [reply]
> 请问验证码签名怎么弄的？

抓包抓到的~

**Reply by KennyZKY** [reply]
已尝试迅雷专家版，挂载成功，一开始正常使用，后面但问题依旧。。目测我网络设置问题？
